Hermes/hermes.py

```diff
@@ -248,18 +276,39 @@ def main():
   logging.info("Hermes is up and listening on port: {0}.".
     format(options.port))
 
-  # Periodically check with the portal for new tasks.
-  # Note: Currently, any active handlers from the tornado app will block
-  # polling until they complete.
-  PeriodicCallback(poll, hermes_constants.POLLING_INTERVAL).start()
+  master = appscale_info.get_private_ip() in \
+           appscale_info.get_load_balancer_ips()
```

Comment:
I don't remember how our discussion concluded, but I'm not sure we need multiple machines performing stats collection for the entire cluster.